.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: c
+sudo: false
+
+cache:
+  directories:
```

Comment:
The Travis docs suggest it's not worth it to cache individual binaries (protoc or stack), since they pull the cached data over the network for each build anyway.
https://docs.travis-ci.com/user/caching#How-does-the-caching-work%3F